docs/plugins/plugin-script.asciidoc

```diff
@@ -0,0 +1,229 @@
+[[plugin-management]]
+== Plugin Management
+
+The `plugin` script is used to install, list, and remove plugins. It is
+located in the `$ES_HOME/bin` directory by default but it may be in a
+{ref}/setup-dir-layout.html[different location] if you installed Elasticsearch
+with an RPM or deb package.
+
+Run the following command to get usage instructions:
+
+[source,shell]
+-----------------------------------
+bin/plugin -h
+-----------------------------------
+
+[[installation]]
+=== Installing Plugins
+
+The documentation for each plugin usually includes specific installation
+instructions for that plugin, but below we document the various available
+options:
+
+[float]
+==== Core Elasticsearch plugins
+
+Core Elasticsearch plugins can be installed as follows:
+
+[source,shell]
+-----------------------------------
+bin/plugin --install [plugin_name]
```

Comment:
We should use here `bin/plugin install [plugin_name]` so we will be compliant to the next version of the plugin manager which will make this mandatory (#9998)